weekly/create_weekly.sh

```diff
@@ -45,7 +45,7 @@ set -e
+( $DRYRUN ) || makePfsDesign --fibers "$FIBERS" --spectrograph 1 --pfsDesignId 1 --scienceCatId 1 --scienceObjId "18 55 71 76 93 94 105 112 115 120 125 139 140 146 148 160 177 178 185 212 215 218 234 242 249 253 277 292 294 304 308 314 329 331 336 342 343 346 358 359 382 387 391 392 396 398 406 416 427 428 429 431 439 440 446 447 449 452 456 462 469 474 476 481 488 492 496 497 499 506 508 534 537 560 565 566 567 571 580 585 586 591 592 596 605 606 607 613 614 615 619 623 624 626 628 632 633 637 645 647 654 674 685 689 700 702 704 724 725 726 727 733 737 741 762 764 765 766 775 783 791 792 797 798 804 809 814 820 823 824 832 833 835 838 849 852 866 867 868 869 879 893 906 939 954 959 961 962 967 975 976 978 983 991 997 1002 1010 1022 1027 1029 1034 1035 1065 1070 1071 1075 1077 1078 1084 1086 1103 1104 1109 1113 1118 1120 1127 1146 1155 1165 1167 1171 1178 1181 1186 1192 1194 1195 1200 1201 1202 1207 1212 1215 1217 1219 1225 1227 1228 1243 1245 1246 1279 1289 1290 1294 1299 1301 1313 1314 1319 1323 1324 1328 1336 1341 1342 1344 1345 1348 1351 1352 1354 1355 1357 1360 1371 1373 1387 1392 1396 1400 1405 1410 1411 1415 1423 1424 1427 1430 1431 1440 1441 1453 1455 1456 1459 1466 1474 1475 1483 1489 1492 1494 1496 1501 1502 1505 1511 1517 1518 1528 1536 1551 1552 1571 1574 1593 1601 1605 1608 1621 1632 1636 1639 1643 1648 1653 1655 1659 1660 1665 1667 1668 1675 1676 1680 1692 1695 1715 1741 1743 1762 1764 1765 1766 1767 1770 1772 1783 1784 1788 1798 1822 1824 1849 1862 1865 1905 1908 1914 1919 1921 1922 1928 1935 1941 1944 1947 1948 1950 1953 1955 1962 1964 1965 1968 1970 1972 1973 1975 1976 1980 1981 1982 1986 1999 2005 2012 2015 2019 2035 2040 2044 2047 2055 2071 2074 2076 2082 2097 2104 2106 2111 2114 2120 2127 2131 2132 2133 2135 2136 2143 2149 2155 2157 2158 2162 2164 2177 2181 2184 2188 2189 2201 2210 2217 2238 2242 2244 2251 2257 2258 2262 2263 2271 2274 2285 2291 2296 2299 2307 2311 2312 2336 2343 2344 2348 2354 2358 2375 2376 2377 2378 2380 2382 2383 2385 2403 2405 2416 2417 2423 2427 2439 2446 2451 2452 2453 2458"
```

Comment:
This is fine for now, but I suspect we will need to run tests of the pipeline against the full set of 4 spectrograph modules in the future. We might need to think about how to do that in the future, eg an option to specifiy ``objIds`` in a separate CSV file.